### PR TITLE
feat(route): 牛客网面经

### PR DESCRIPTION
**Pull request by huyyi**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/nowcoder/experience/:tagId
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

## 说明 / Note
添加牛客网某职位的面经RSS，提取自网站接口json

### CODE REVIEW

File: lib/v2/nowcoder/experience.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    const response = await got.get(link.toString());
+    const data = response.data.data;
+
+    const list = data.discussPosts
+        .map((x) => {
+            const info = {
+                title: x.postTitle,
+                link: new URL('discuss/' + x.postId, host).href,
+                author: x.author,
+                pubDate: timezone(parseDate(new Date(x.createTime), +8)),
```
Comment (TonyRL) [inline lib/v2/nowcoder/experience.js:25]:
```suggestion
                pubDate: timezone(parseDate(x.createTime), +8),
```

File: docs/bbs.md [hunk 2]
```diff
@@ -524,6 +524,14 @@ pageClass: routes
...
 ## 牛客网
 
+### 面经
+<Route author="huyyi" example="/nowcoder/experience/639?order=3&companyId=665&phaseId=0" path="/nowcoder/experience/:tagId" :paramsDesc="['职位id [查询链接](https://www.nowcoder.com/profile/all-jobs)']">
+可选参数：
+- companyId：筛选公司，公司的id可以[查询](https://www.nowcoder.com/discuss/tag/exp)
+- order：3-最新；1-最热
+- phaseId：0-所有；1-校招；2-实习；3-社招
+</Route>
+
```
Comment (TonyRL) [inline docs/bbs.md:534]:
```suggestion
### 面经

<Route author="huyyi" example="/nowcoder/experience/639?order=3&companyId=665&phaseId=0" path="/nowcoder/experience/:tagId" :paramsDesc="['职位id [查询链接](https://www.nowcoder.com/profile/all-jobs)']">

可选参数：
- companyId：筛选公司，公司的id可以[查询](https://www.nowcoder.com/discuss/tag/exp)
- order：3-最新；1-最热
- phaseId：0-所有；1-校招；2-实习；3-社招

</Route>

```

**Comment by TonyRL** [comment]
example `/nowcoder/experience/639?order=3&companyId=665&phaseId=0` does not work but `/nowcoder/experience/639` is working

**Comment by huyyi** [comment]
> example `/nowcoder/experience/639?order=3&companyId=665&phaseId=0` does not work but `/nowcoder/experience/639` is working

This problem fixed, I return URL object by mistake.😄

**Pull request merged**